roles/openshift_version/tasks/main.yml

```diff
@@ -48,17 +78,35 @@
   include: set_version_containerized.yml
   when: is_containerized | bool
 
+# Stop if the user has provided an openshift_image_tag but is not doing a containerized install
+- assert:
+    that: openshift_image_tag is not defined
+    message: >
+      openshift_image_tag is used for containerized installs. If you are trying to
+      specify an image for a non-container install see oreg_url.
+  when: not is_containerized | bool
```

Comment:
Right now the ``openshift_image_tag`` is only honored during containerized installs. I'll add a note that this will have to be removed/modified with the updates to container + rpm installs.